src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -64,13 +77,36 @@ const innerJoinIngredients = `LEFT JOIN PIZZA_INGREDIENTS ON PIZZAS.ID = PIZZA_I
  * A query to fetch a pizza with, optionally, its ingredients and doughs
  *
  * @param {string[]} included
+ * @param {string} conditionals
  * @returns {string}
  */
-const getPizzaByIdQuery = (included) => dedent`SELECT ${getColumnAliases(included).join(',\n    ')}
+const getPizzaByIdQuery = (included, conditionals) => dedent`SELECT ${getColumnAliases(included).join(',\n    ')}
   FROM PIZZAS
   ${included.includes('dough') ? innerJoinDoughs : ''}
   ${included.includes('ingredients') ? innerJoinIngredients : ''}
-  WHERE PIZZAS.ID = :id`;
+  ${conditionals ? `WHERE ${conditionals}` : ''}`;
+
+/**
+ * get conditionals and bind params for a SELECT query
+ * to get multiple pizzas from the database
+ *
+ * @param {object} query the query parameters passed to the HTTP request
+ * @returns {object} `bindParams` (object) and `conditionals` (string)
+ */
+const getConditionalsAndBindParams = (query) => {
+  const conditionalStatements = [];
+  const bindParams = {};
+  _.forEach(query, (parameterValue, parameterName) => {
+    if (pizzaFilters.includes(parameterName)
+      && parameterName.match(/filter\[.*\]/)) {
```

Comment:
`pizzaFilters` is named badly, so it also includes parameters like `include` that don't match the regex. I think I'm going to add a `filter()` call when `pizzaFilters` is generated so that I can remove the check.